android/titanium/src/java/org/appcelerator/titanium/view/TiBorderWrapperView.java

```diff
@@ -121,6 +123,14 @@ private void updateBorderPath()
 			borderPath.addRect(innerRect, Direction.CCW);
 			borderPath.setFillType(FillType.EVEN_ODD);
 		}
+		// set the outline for the view in order to use elevation
+		ViewOutlineProvider viewOutlineProvider = new ViewOutlineProvider() {
```

Comment:
ViewOutlineProvider is an API 21+ only, please add an API check here. Also, this won't work on API 20 and below.